nautobot/core/ui/object_detail.py

```diff
@@ -164,6 +165,9 @@ def local_context(self, context, **kwargs):
             context = context.flatten()
         return {**context, **kwargs}
 
+    def get_obj_from_context(self, context):
+        return context.get("obj") or context.get("object")
+
```

Comment:
Redundant now with similar `get_obj_from_context` utility function added in #6140?